nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/api/RegistryAboutResource.java

```diff
@@ -56,8 +59,13 @@ public RegistryAboutResource(
             response = RegistryAbout.class
     )
     public Response getVersion() {
-        final String implVersion = NiFiRegistryApiApplication.class.getPackage().getImplementationVersion();
-        final RegistryAbout version = new RegistryAbout(implVersion);
-        return Response.status(Response.Status.OK).entity(version).build();
+        Properties props = new Properties();
+        try (InputStream stream = getClass().getResourceAsStream(PROPERTIES_FILE)){
+            props.load(stream);
+        } catch (IOException e) {
+            return Response.status(Response.Status.OK).entity("unknown").build();
```

Comment:
Should we log that an error occurred here so it would be possible to triage why "unknown" appears as a version?